cloud/aws/cli/plugins/misp.py

```diff
@@ -0,0 +1,86 @@
+import os
+import pathlib
+import stat
+from typing import Tuple
+from vast_invoke import Context, pty_task, task
+import dynaconf
+from common import (
+    FargateService,
+    aws,
+    terraform_output,
+)
+
+VALIDATORS = [dynaconf.Validator("VAST_MISP_VERSION", default="v2.4.145a")]
+
+
+def service_outputs(c: Context) -> Tuple[str, str, str]:
+    cluster = terraform_output(c, "core-2", "fargate_cluster_name")
+    family = terraform_output(c, "misp", "misp_task_family")
+    service_name = terraform_output(c, "misp", "misp_service_name")
+    return (cluster, service_name, family)
+
+
+@task
+def status(c):
+    """Get the status of the MISP service"""
+    print(FargateService(*service_outputs(c)).get_task_status())
+
+
+@task
+def start(c):
+    """Start the MISP as an AWS Fargate task. Noop if it is already running"""
+    FargateService(*service_outputs(c)).start_service()
+
+
+@task
+def stop(c):
+    """Stop the MISP instance and service"""
+    FargateService(*service_outputs(c)).stop_service()
+
+
+@task
+def restart(c):
+    """Stop the running MISP task, the service starts a new one"""
+    FargateService(*service_outputs(c)).restart_service()
+
+
+def get_public_ip(c):
+    """Resolve the public IP of the MISP task"""
+    task = FargateService(*service_outputs(c)).describe_task()
+    eni_attach = next(
+        x for x in task["attachments"] if x["type"] == "ElasticNetworkInterface"
+    )
+    eni_id = next(
+        x["value"] for x in eni_attach["details"] if x["name"] == "networkInterfaceId"
+    )
+
+    eni_desc = aws("ec2").describe_network_interfaces(NetworkInterfaceIds=[eni_id])
+    return eni_desc["NetworkInterfaces"][0]["Association"]["PublicIp"]
+
+
+@task
+def tunnel(c, ui_port="8080", zmq_port="50000"):
```

Comment:
Tunnel seems to be not successful after deploy. No error messages.

![image](https://user-images.githubusercontent.com/5330275/189080080-789e4511-2522-4f57-9a97-d57ad6604959.png)

Reply:
I am wondering why no error message is bubbling up here. The issue is that you didn't start MISP

Reply:
I made the message a bit more explicit (`No task running`)